tailwind.config.js

```diff
@@ -2,7 +2,14 @@ module.exports = {
   purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      backgroundImage: {
+        // 'main-image': "url('./img/placeholder.jpg')",
+        'main-image': `url('${
```

Comment:
Hi @timamero! I was able to get the image to load by adjusting the file path on line 7 to:
`'main-image': "url('/src/img/placeholder.jpg')",`. With this change the duplicate image in the public folder can be removed.